ouroboros-node/ouroboros-node.cabal

```diff
@@ -0,0 +1,96 @@
+cabal-version:       3.0
+name:                ouroboros-node
```

Comment:
Would you update the PR description to say `ouroboros-node` instead of `ouroboros-consensus-node`?

Reply:
There's no description, but also there's no `ouroboros-node` anywhere.

Reply:
By "PR description" I meant the thing at the top of this PR page. It looks like this to me:

![image](https://user-images.githubusercontent.com/1106131/203626787-42f5694c-9cc4-40df-babf-b5eb8b42b35a.png)

Reply:
The full graph is >10MB and is refused by github.